tests/process/spawn/stream.luau

```diff
@@ -12,7 +12,7 @@ assert(
 )
 
 local echoChild = if process.os == "windows"
-	then process.spawn("echo", { msg, "1>&2" }, { shell = "cmd" })
+	then process.spawn("cmd.exe", { "/c", "echo", msg, "1>&2" })
```

Comment:
With your method
```lua
then process.spawn("cmd.exe", { "/c", "echo", msg, "1>&2" })
```
I got the message but wrapped with quotes, would look like this `"hello, world"`.

With the method I provided on the last review comment.
```lua
then process.spawn("/c", { "echo", msg, "1>&2" }, { shell = "cmd" })
```
the stderr was exactly the message, would look like this `hello, world`.

Its only whatever matters format wise, as my method seems more hacky imo.

Reply:
Thank you! Fixed in https://github.com/lune-org/lune/pull/211/commits/cf707a37ff5d8ed9b6fef66e4640d89be30a2217.